### PR TITLE
add-form

### PR DESCRIPTION
**Pull request by mumbereh**
I this branch I have added the form section to complete mobile portfolio. 
The form is capable to capture data and submit it to the my email.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -442,6 +442,7 @@ image {
   color: #868a8f;
   width: 87%;
 }
+
 .form {
   background: #6070ff;
   background-image: url(contact-bg.png);
```
Comment (Owusu-Desmond) [inline style.css]: - [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design so make sure that you adjust the styling (the background image alignment of the form) to look exactly like the Figma design. :pray: 

| Your Design |Figma Design |
| :---:   | :---: | 
| image | image|

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @mumbereh and @DevKaranJ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Nice UI design and clean code :+1: ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (shella12) [inline index.html:1]: - [ ] [OPTIONAL] Please make sure all form fields have attribute `required` including `textarea`  as according to the requirements:

> * Make sure that:
>     - A user cannot submit the form if one or more of the fields is empty.

**Review by shella12 — APPROVED** [review]
Hi @mumbereh, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great design implementation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mumbereh** [comment]
Thank you all my code reviewers. 
You made it for me to this success.

**Pull request merged**